Vault-Deployment/terraform.tfvars.example

```diff
@@ -17,4 +17,7 @@ prometheus-vault-access-token = ""
 organizationName       = "TestOrg"
 organizationalUnitName = "Devops"
 commonName             = "vault-internal.svc.cluster.local"
-additionalDomains      = ["vault.cluster.local", "vault-cli.playground.lab", "vault-ui.playground.lab", "vault.default.svc.cluster.local", "vault-0.vault-internal", "vault-1.vault-internal", "vault-2.vault-internal"]
+additionalDomains      = ["vault.cluster.local", "vault.playground.lab", "vault.default.svc.cluster.local", "vault-0.vault-internal", "vault-1.vault-internal", "vault-2.vault-internal"]
```

Comment:
@Xelef2000 simplifies the Vault API naming here by relying only on `vault.playground.lab`. We can use the same API name for UI/CLI purposes.

Reply:
With everywhere connecting to vault using `vault.playground.lab` I think this should be the only domain in the TLS certificate

Reply:
https://github.com/adfinis/vault-playground-on-kubernetes/issues/25